content/en/user-guide/tools/lambda-tools/lambda-vscode-extension/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Lambda VSCode Extension"
+weight: 9
+description: >
+  Deploy and invoke Lambda functions in LocalStack directly from VSCode
+---
+
+## Introduction
+
+[LocalStack Lambda VSCode Extension](https://github.com/localstack/localstack-vscode-extension) supports deploying and invoking Python Lambda functions through [AWS SAM](https://github.com/aws/serverless-application-model) or [AWS CloudFormation](https://aws.amazon.com/cloudformation/resources/templates/).
+
+## Prerequisites
+
+- [VS Code](https://code.visualstudio.com/)
+- [`samlocal`](https://github.com/localstack/aws-sam-cli-local) command line wrapper around the [AWS SAM CLI](https://github.com/aws/aws-sam-cli) for use with [LocalStack](https://github.com/localstack/localstack).
+- [LocalStack](https://docs.localstack.cloud/getting-started/) running in the background.
+
+## Getting Started
+
+You can use a [sample project](https://github.com/joe4dev/lambda-python) to get started with the extension. The sample project contains a simple Lambda function and a SAM template. Clone the repository and open the project in VSCode.
```

Comment:
It would be generally better to refer to LocalStack-owned repositories but I see that it might be easiest to just leave it for this small sample ...